bin/server_package_standalone

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This is identical to bin/client_package_standalone except for binary name. Couldn't we use a single script for both of these?

Reply:
If not, then could we at least start to tidy up the file structure in bin/ separate the node from client scripts?

Reply:
Soon we will migrate to single binary `mysterium_node`, so I care about it much ATM